app/views/catarse_bootstrap/projects/_dashboard_goal.html.slim

```diff
@@ -18,18 +18,27 @@
                 preppend_input_text: 'R$',
                 preppend_input_text_wrapper_html: { class: 'fontsize-base lineheight-tightest' }
 
-              = form.input :online_days,
-                type: 'number',
-                wrapper: :two_columns_with_postfix,
-                wrapper_class: 'w-row u-marginbottom-30 card card-terciary medium',
-                label_wrapper_html: { class: 'w-col w-col-5 w-subcol' },
-                text_field_wrapper_html: { class: 'w-col w-col-7' },
-                input_html: { class: 'prefix positive medium', required: false },
-                field_wrapper_html: { class: 'w-col w-col-8 w-col-small-6 w-col-tiny-6' },
-                validation_text: false,
-                append_text_wrapper_html: { class: 'w-col w-col-4 w-col-small-6 w-col-tiny-6 text-field postfix no-hover medium prefix-permalink' },
-                append_input_text: 'Dias',
-                append_input_text_wrapper_html: { class: 'fontsize-base lineheight-tightest' }
+              = render "#{@project.mode}_dashboard_explanation"
+
+              - if @project.is_flexible?
+                .w-row.u-marginbottom-30.card.card-terciary.medium
+                  .w-col.w-col-5.w-sub-col._w-sub-col
+                    label.w-editable.fontweight-semibold.fontsize-base= t(".#{@project.mode}.expiration_title")
+                  .w-col.w-col-7
+                    .w-editable.fontsize-small= t(".#{@project.mode}.expiration_text")
```

Comment:
This mode logic is duplicated with the above if.
One solution is to use hardcode the I18n keys or create two separate partials.